src/theme/StatBox.ts

```diff
@@ -0,0 +1,33 @@
+export const StatBox = {
+  baseStyle: {
+    stack: {
+      background: "gray.800",
+      padding: 8,
+      borderRadius: "md",
+    },
+    mainText: {},
+    secondaryText: {
+      color: "gray.500",
+    },
+  },
+  variants: {
+    primary: {
+      stack: {
+        backgroundColor: "gray.900",
+      },
+    },
+    secondary: {
+      stack: {
+        backgroundColor: "#181D23",
```

Comment:
Can we define this color in `theme/index.ts` in `colors` variable?

Reply:
3c23836d0ee484230c2848f228a1f79c9848de4e